server/src/models/database/socobouser.ts

```diff
@@ -0,0 +1,10 @@
+export class SocoboUser {
```

Comment:
Good idea. I add this in another issue. We should complete this to move forward. This PR goes into branch 12 user management and not directly into the master.